catroid/src/org/catrobat/catroid/devices/arduino/phiro/PhiroImpl.java

```diff
@@ -0,0 +1,384 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2015 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.devices.arduino.phiro;
+
+import android.util.Log;
+
+import org.catrobat.catroid.bluetooth.base.BluetoothConnection;
+import org.catrobat.catroid.bluetooth.base.BluetoothDevice;
+import org.catrobat.catroid.formulaeditor.Sensors;
+
+import java.io.IOException;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.UUID;
+
+import name.antonsmirnov.firmata.Firmata;
+import name.antonsmirnov.firmata.message.AnalogMessage;
+import name.antonsmirnov.firmata.message.Message;
+import name.antonsmirnov.firmata.message.ReportAnalogPinMessage;
+import name.antonsmirnov.firmata.message.ReportFirmwareVersionMessage;
+import name.antonsmirnov.firmata.message.SetPinModeMessage;
+import name.antonsmirnov.firmata.serial.ISerial;
+import name.antonsmirnov.firmata.serial.SerialException;
+import name.antonsmirnov.firmata.serial.StreamingSerialAdapter;
+
+public class PhiroImpl implements Phiro {
+
+	private static final UUID PHIRO_UUID = UUID.fromString("00001101-0000-1000-8000-00805F9B34FB");
```

Comment:
No, it is a bluetooth uuid. This is the default one. Also Lego, Arduino, Albert, etc. use it.